server/models/user-model.js

```diff
@@ -12,3 +12,36 @@ const Schema = mongoose.Schema;
 // );
 
 // module.exports = mongoose.model("messages", Message);
+
+const Setting = new Schema({
+  _id: { type: String, required: true },
+  notification: { type: Boolean, required: true },
+  darkMode: { type: Boolean, required: true },
+  language: { type: String, required: true },
+  disableAcc: { type: Boolean, required: true },
+});
+
+const Post = new Schema({
+  _id: { type: String, required: true }, // TODO: check correct type for objectid
```

Comment:
we're using the Spotify username which is a string so this is all good!

Reply:
thanks!